pkg/asset/cluster/cluster.go

```diff
@@ -68,11 +65,24 @@ func (c *Cluster) Generate(parents map[asset.Asset]*asset.State) (*asset.State,
 		return nil, fmt.Errorf("failed to unmarshal terraform tfvars file: %v", err)
 	}
 
-	templateDir, err := terraform.FindStepTemplates(dir, terraform.InfraStep, tfvars.Platform)
+	if err := data.Unpack(tmpDir); err != nil {
+		return nil, err
+	}
+
+	templateDir, err := terraform.FindStepTemplates(tmpDir, terraform.InfraStep, tfvars.Platform)
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, errors.New("infra step not found; set OPENSHIFT_INSTALL_DATA to point to the data directory")
+		}
 		return nil, fmt.Errorf("error finding terraform templates: %v", err)
 	}
 
+	// take advantage of the new installer only having one step.
+	err = os.Rename(filepath.Join(tmpDir, "config.tf"), filepath.Join(templateDir, "config.tf"))
```

Comment:
shouldn't this happen in `Unpack`?

Reply:
> shouldn't this happen in `Unpack`?

`Unpack` is generic, just dump the `http.FileSystem` to disk.  This is specific to the content that we happen to stick into `Unpack`.  And I was going to shuffle this around later, and not bother to write the step that we don't need (e.g. only unpack `modules` and `steps/infra/aws` for AWS).  But I hadn't worked that up yet.